recruitment/views.py

```diff
@@ -180,10 +180,15 @@ def __init__(self, description, monocolor, charts, values=None, sort_key=None):
             self.monocolor = monocolor
             self.charts = charts
             self.sort_key = sort_key
+            self.y_limit = None
 
             if values:
                 self.add_values(values)
 
+
+        def set_y_limit(self, y_limit):
```

Comment:
If you really need to use a setter (the instance variables are public anyway) please rather use the `@property` decorator together with `y_limit`.